libs/api/domains/content-search/src/lib/contentSearch.resolver.ts

```diff
@@ -8,6 +8,11 @@ import { WebSearchAutocomplete } from './models/webSearchAutocomplete.model'
 @Resolver()
 export class ContentSearchResolver {
   constructor(private contentSearchService: ContentSearchService) {}
+  // trigger
```

Comment:
this will be removed in a final cleanup - this is a hack to trigger builds for feature-deploy - infra will fix and make this unnecessary hopefully soon